app/services/feature_flag.rb

```diff
@@ -33,6 +33,7 @@ def feature
     [:sample_applications_factory, 'An alternate generator for test/sample applications, uses `SampleApplicationsFactory` in place of `TestApplications.new`', 'Elliot Crosby-McCullough + Tomas Destefi'],
     [:structured_reference_condition, 'Structured reference condition that can be added as a condition to an offer', 'Tomas Destefi'],
     [:continuous_applications, 'The new continuous applications flow', 'James Glenn'],
+    [:international_relocation_payment, 'Shows international relocation information if candidate is eligible', 'Lori Bailey'],
```

Comment:
Are feature flags added in the default `active?` or `inactive?` state?
This feature flag would ideally be added as `active?` unless we renamed it to `:remove_international_relocation_payment` - all the double negatives 🙈

Reply:
Hi David, Feature Flags are inactive as default, so I think we should reverse the language so that we hide the relocation information when active. 

It feels a bit like an anti-pattern because feature are generally for hiding a new feature, not removing one. I think that makes more sense. I'll move this back to draft, change it around and then ask for another review.

Reply:
Hey - we've literally just been talking about this and we're happy to skip the feature flag and remove the functionality - this means we'd need to leave these PRs open until we have confirmation 👌